contrib/razor/template.py

```diff
@@ -0,0 +1,16 @@
+pkgname = "razor"
```

Comment:
should likely be called perl-razor2-client-agent

Reply:
that is, unless the perl module is secondary and the primary thing is the frontend tools

Reply:
some distros call it razor and some call it that:
https://repology.org/project/perl:razor2-client-agent/versions

Reply:
changing the name brings back the previous build failure?

Reply:
fixed it with another upstream patch, was parallel make

Reply:
i still don't really understand how this is supposed to be used

so it is a perl module, is it a program, or what

Reply:
it is a program written in perl, it installs in user bin:
razor-admin   razor-check   razor-client  razor-report  razor-revoke

but also installs perl modules in /usr/lib/perl5 that these use

Reply:
if it's primarily a program, calling it razor is fine

Reply:
okay changed back, yes it a program for checking email hashes against a database for spam